local.sh

```diff
@@ -157,7 +157,7 @@ EOF
     echo "Locust application is successfully deployed. you can access http://<docker-host-ip-address>:8089"
 
     if [[ "$MODE" =~ ^(automatic|Automatic|auto)$ ]]; then
-        sleep 5
+        sleep 9
```

Comment:
why do you increase the sleep time?

Reply:
Because the Controller has a delay of 4 seconds + locust startup wait time + downloading delay. It can probably be lowered by 1 more second - let me check